scripts/Muon/GUI/Common/contexts/plot_settings_context.py

```diff
@@ -10,9 +10,17 @@ class PlotSettingsContext(object):
     def __init__(self):
         self._min_y_range = 2.0
         self._y_axis_margin = 20.
+        self._x_axis_margin = 10.0
         self._is_tiled = False
         self._is_tiled_by = ""
         self._is_condensed = False
+        self._linestyle = {}
+        self._default_linestyle = "-"
+        self._marker = {}
+        self._default_marker = ""
+        self._wrap_width = 30# determines the width of text on axis ticks
+        self._font_size = "xx-small"
```

Comment:
Can the size of the text be changed later by the user? Also does hard coding the size like this mean it doesn't scale well on different displays?

Reply:
yes, so that will be part of plot style in the plot fine tuning